src/formats/mmCIF.cpp

```diff
@@ -190,6 +202,14 @@ void mmCIFFormat::read(Frame& frame) {
     frame.resize(0);
     residues_.clear();
 
+    if (name_.size()) {
+        frame.set("name", name_);
+    }
+
+    if (pdb_idcode_.size()) {
```

Comment:
Could you use `!pdb_icode.empty()` instead? It is a bit clearer. Or even `pdb_icode != ""`

Reply:
That makes too much sense 😸